package.json

```diff
@@ -33,20 +33,29 @@
     "tap"
   ],
   "dependencies": {
+    "dargs": "^5.1.0",
+    "execa": "^0.6.0",
     "gulp-util": "^3.0.0",
-    "mocha": "^3.0.0",
-    "plur": "^2.1.0",
-    "req-cwd": "^1.0.1",
-    "temp": "^0.8.3",
-    "through": "^2.3.4"
+    "through2": "^2.0.3"
   },
   "devDependencies": {
+    "mocha": "^3.0.0",
```

Comment:
ah. I had thought the ecosystem was moving towards the pattern that gulp-jshint is using now; where jshint is a peerDep. Will remedy.